astropy/modeling/core.py

```diff
@@ -2315,6 +2409,95 @@ def _custom_model_wrapper(func, fit_deriv=None):
     return type(model_name, (FittableModel,), members)
 
 
+def render_model(model, arr=None, coords=None):
```

Comment:
Are there still plans to make this a method of `Model` instead?  I see no reason that it shouldn't be.